.github/workflows/post-integration.yml

```
@@ -40,11 +40,14 @@ jobs:
         with:
           java-version: 1.11
 
-      - name: 📋 Prepare build for unit tests
-        run: |
-          dotnet clean -c Debug
-          dotnet restore
-          dotnet build -c Debug /p:UseSourceLink=true --no-restore
+      - name: 🧹 Clean
+        run: dotnet clean -c Release && dotnet nuget locals all --clear
+
+      - name: 🔁 Restore packages
+        run: dotnet restore
+
+      - name: 🛠️ Build
+        run: dotnet build -c Release --no-restore /p:UseSourceLink=true
```

```suggestion
        run: dotnet build -c Release --no-restore /p:UseSourceLink=true /p:ContinuousIntegrationBuild=true
```

Pretty sure it is best practise to set the `p:ContinuousIntegrationBuild=true` during build or `dotnet pack`.

---

This is part of the `Directory.Build.props` file under `src`.